<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1177</width>
    <height>735</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>9</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Fiyat Çekme ve Karşılaştırma</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>20</y>
      <width>151</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Karşılaştırma Tablosu:</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>60</y>
      <width>851</width>
      <height>581</height>
     </rect>
    </property>
    <row>
     <property name="text">
      <string>1</string>
     </property>
    </row>
    <column>
     <property name="text">
      <string>Ürün Kodu</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Ürün Adı</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Barkod</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Eski Fiyat</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Yeni Fiyat</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Para Birimi</string>
     </property>
    </column>
    <item row="0" column="0">
     <property name="text">
      <string/>
     </property>
     <property name="flags">
      <set>NoItemFlags</set>
     </property>
    </item>
    <item row="0" column="1">
     <property name="text">
      <string/>
     </property>
     <property name="flags">
      <set>NoItemFlags</set>
     </property>
    </item>
    <item row="0" column="2">
     <property name="text">
      <string/>
     </property>
     <property name="flags">
      <set>NoItemFlags</set>
     </property>
    </item>
    <item row="0" column="3">
     <property name="text">
      <string>asd</string>
     </property>
     <property name="background">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>255</green>
        <blue>0</blue>
       </color>
      </brush>
     </property>
     <property name="foreground">
      <brush brushstyle="NoBrush">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </property>
     <property name="flags">
      <set>NoItemFlags</set>
     </property>
    </item>
    <item row="0" column="4">
     <property name="text">
      <string>ssss</string>
     </property>
     <property name="background">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </property>
     <property name="foreground">
      <brush brushstyle="NoBrush">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </property>
     <property name="flags">
      <set>NoItemFlags</set>
     </property>
    </item>
    <item row="0" column="5">
     <property name="text">
      <string/>
     </property>
     <property name="flags">
      <set>NoItemFlags</set>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="btnComparisontoExcel">
    <property name="geometry">
     <rect>
      <x>1022</x>
      <y>20</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Excele Aktar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnComparison">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>20</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Karşılaştırmayı Başlat</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblProgressInfo">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>660</y>
      <width>1151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>İşlem Bilgisi: </string>
    </property>
   </widget>
   <widget class="QTextEdit" name="txtPassword">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>204</y>
      <width>256</width>
      <height>32</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>180</y>
      <width>28</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Şifre</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnStart">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>600</y>
      <width>251</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>BAŞLA</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cmbTableType">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>23</y>
      <width>251</width>
      <height>41</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Gelişmiş Tablo</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Detaylı Tablo</string>
     </property>
    </item>
   </widget>
   <widget class="QCheckBox" name="chkImageDownload">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>291</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Resimleri İndir (İşlem Daha Uzun Sürer)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>116</y>
      <width>72</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Kullanıcı Adı</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textUsername">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>256</width>
      <height>32</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lblPageLen">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>540</y>
      <width>251</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lblItemLen">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>570</y>
      <width>251</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1177</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
